frontend/packages/operator-lifecycle-manager/src/components/operand/useShowOperandsInAllNamespaces.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector, useDispatch } from 'react-redux';
+import * as UIActions from '@console/internal/actions/ui';
+import { RootState } from '@console/internal/redux';
+
+type ShowOperandsInAllNamespaces = {
+  showOperandsInAllNamespaces: boolean;
+  setShowOperandsInAllNamespaces: (value: boolean) => void;
+};
+// This hook can be used to consume and update the showOperandsInAllNamespaces redux state
+export const useShowOperandsInAllNamespaces = (): ShowOperandsInAllNamespaces => {
+  const dispatch = useDispatch();
+
+  const showOperandsInAllNamespaces = useSelector((state: RootState) =>
+    state.UI.get('showOperandsInAllNamespaces'),
+  );
+
+  const setShowOperandsInAllNamespaces = React.useCallback(
+    (value: boolean) => dispatch(UIActions.setShowOperandsInAllNamespaces(value)),
+    [dispatch],
+  );
+
+  return { showOperandsInAllNamespaces, setShowOperandsInAllNamespaces };
+};
```

Comment:
Convention is to return a tuple from a hook:

```suggestion
type UseShowOperandsInAllNamespaces = () => [boolean, (value: boolean) => void];
// This hook can be used to consume and update the showOperandsInAllNamespaces redux state
export const useShowOperandsInAllNamespaces: UseShowOperandsInAllNamespaces = () => {
  const dispatch = useDispatch();

  const showOperandsInAllNamespaces = useSelector((state: RootState) =>
    state.UI.get('showOperandsInAllNamespaces'),
  );

  const setShowOperandsInAllNamespaces = React.useCallback(
    (value: boolean) => dispatch(UIActions.setShowOperandsInAllNamespaces(value)),
    [dispatch],
  );

  return [showOperandsInAllNamespaces, setShowOperandsInAllNamespaces];
};
```